README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
 
 What's the difference? Well consider a mechanic driving to work. In that
 moment, waiting in traffic, they are a _user_ of the car. But then they take a
-right turn and *cough*, *splutter* their car chokes to a stop. At once their
-mind changes to being a _mechanic_ or a _developer_. They think about
+right turn and *cough*, *splutter* &mdash; their car chokes to a stop. At once
+their mind changes to being a _mechanic_ or a _developer_. They think about
 petroleum, spark-plugs and ignitions. They can _debug_ their car using
 techniques they've learned. They can imagine what might be wrong. To get more
 data or to verify their guess, they _open up the hood_. Learning to be a
 developer (_computer mechanic_?) is similar. You must start "looking under the
 hood" of your computer. To access the "hood's" contents, you must use the
-shell.
+terminal.
 
-Developers "talk" to the computer by engaging it at the shell level. The shell
+Developers "talk" to their computers through the terminal. The terminal
 is just another way to collaborate with the computer. It might feel
 "old-school" or clunky at first compared with Windows or MacOSX. As you learn
 more, though, it might surprise you!
 
-Shell skills are critical in being a developer or HTML author.  Comfort in the
-shell is an assumption for many technical jobs.
+Terminal skills are critical in being a developer or HTML author.  Comfort in the
+terminal is an assumption for many technical jobs.
```

Comment:
I'm questioning the word assumption here (pedantic, I know). Would requirement feel better? I'm really not sure, but it feels like it might.